pkgs/development/python-modules/gradio-pdf/default.nix

```diff
@@ -11,16 +11,16 @@
 
 buildPythonPackage rec {
   pname = "gradio-pdf";
-  version = "0.0.15";
+  version = "0.0.17";
   format = "pyproject";
 
   src = fetchPypi {
     pname = "gradio_pdf";
     inherit version;
-    hash = "sha256-R+4CDJo84/+PUTTxoz9CsZlHUVsldgjufWsRkALrfN4=";
+    hash = "sha256-LoVcwE7eGcK5Nc6qKTnrnI+rNlsDbekhKUP+Fzq2SQ8=";
   };
```

Comment:
Have you tried using the github repo instead ?

Reply:
no tags

Reply:
Ok :'(, thanks for trying.